AEPIdentity/Tests/IdentityStateTests.swift

```diff
@@ -868,6 +879,8 @@ class IdentityStateTests: XCTestCase {
         // test
         state.processPrivacyChange(event: event, createSharedState: { _, _ in
             sharedStateExpectation.fulfill()
+        }, createXDMSharedState: { _, _ in
+            xdmSharedStateExpectation.fulfill()
```

Comment:
We should also have some e2e tests to verify the values stored in the xdm shared state are correct after syncIdentifiers call or setPrivacyStatus changes, I am ok if you want to add that in a separate pr.

Reply:
Already merged :)
I'll get these additional tests in when we determine the final format of the namespace.